src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc_utils.cpp

```diff
@@ -77,8 +83,13 @@ DnnlBlockedMemoryDesc MemoryDescUtils::convertToDnnlBlockedMemoryDesc(const Infe
         std::fill(strides.begin(), strides.end(), 0);
     }
 
-    return DnnlBlockedMemoryDesc(desc.getPrecision(), Shape(desc.getDims()), blkDesc.getBlockDims(), blkDesc.getOrder(), blkDesc.getOffsetPadding(),
-                                 blkDesc.getOffsetPaddingToData(), strides);
+    return DnnlBlockedMemoryDesc(InferenceEngine::details::convertPrecision(desc.getPrecision()),
```

Comment:
I see use of `DnnlBlockedMemoryDesc` with mixed IE precision and ov element types in different places. With current implementation function operates with ov element types. Please, recheck

Reply:
I think we need decouple precision and MemoryDesc, we have another PR to solve TensorDesc issue(https://github.com/openvinotoolkit/openvino/pull/21002), the second merged PR will solve this mixed issue.

Reply:
Agree with @riverlijunjie, will clear this after PR (https://github.com/openvinotoolkit/openvino/pull/21002)